neqo-transport/src/connection/mod.rs

```diff
@@ -267,7 +267,7 @@ pub struct Connection {
     new_token: NewTokenState,
     stats: StatsCell,
     qlog: NeqoQlog,
-
+    last_token: Option<ResumptionToken>,
```

Comment:
I don't think that we need to hold anything here.  If we do need to hold some state so that we can match up tickets with NEW_TOKEN, let's make a new object to manage it.